components/tag/TagSimple.js

```diff
@@ -1,6 +1,6 @@
-export default function TagSimple({ name }) {
+export default function TagSimple({ name, isSelected }) {
   return (
-    <div className="text-sm px-1 mx-px mb-2 font-mono border rounded-md line-clamp-1">
+    <div className={`text-sm px-1 mx-px mb-2 font-mono border rounded-md line-clamp-1 ${isSelected && "border-tertiary-medium"}`}>
```

Comment:
I'm not sure about the implementation steps. Could you provide a bit more detail or even a brief example of how I could integrate this utility function into my existing code? This would greatly help me apply it correctly.

Reply:
Yes, sure. You can find the example using this function in `\pages\roadmap.js` component